l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -874,7 +874,7 @@ def invoiceCreate(
             'currency_id': currency[0].id,
             'journal_id': purchase_journal.id,
             # 'origin': xmlData.datiOrdineAcquisto,
-            'fiscal_position_id': False,
+            'fiscal_position_id': (partner.property_account_position_id.id or False),
```

Comment:
```suggestion
            'fiscal_position_id': partner.property_account_position_id.id,
```
L'`id` dovrebbe essere già `False` se non è settata la posizione fiscale del partner.
Inoltre le parentesi sono ridondanti.

Reply:
si vero, ho tenuto la compatibilità con:
https://github.com/OCA/l10n-italy/blob/efd84ca53e3b7ec2571d2dcb03a2fcf59ba7bb30/l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py#L882

Reply:
modificato come da suggerimento ..

Reply:
mi chiedo, è possibile che referenziando l'`id` in `partner.property_account_position_id.id` dia errore se `partner.property_account_position_id` è`False`?

Reply:
Se il partner non ha la posizione fiscale impostata, `partner.property_account_position_id` dovrebbe essere un recordset vuoto (ad esempio il risultato di `self.env['account.fiscal.position'].browse()`) ma non `False`.
Non ne sono del tutto sicuro perché al momento non lo posso verificare ma direi che lo puoi verificare abbastanza facilmente in locale.

Reply:
Hai ragione! Thx